src/core/component.js

```diff
@@ -31,11 +31,20 @@ var upperCaseRegExp = new RegExp('[A-Z]+');
  *         mapped attribute of the component plus applying defaults and mixins.
  */
 var Component = module.exports.Component = function (el, attrValue, id) {
+  var self = this;
   this.el = el;
   this.id = id;
   this.attrName = this.name + (id ? '__' + id : '');
   this.el.components[this.attrName] = this;
   this.updateProperties(attrValue);
+  this.throttledEmitComponentChanged = utils.throttle(function (oldData) {
+    el.emit('componentchanged', {
+      id: self.id,
+      name: self.name,
+      newData: self.getData(),
+      oldData: oldData
+    }, true);
+  }, 200);
```

Comment:
In time sensitive applications is better to poll instead of relying on events. If you want fresh data I would call `getAttribute` on the event handler.